src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -22,21 +28,23 @@ import {
 import { Helper } from "../../Components/Helper"
 import { TransactionSummaryFragmentContainer as TransactionSummary } from "../../Components/TransactionSummary"
 
-export interface PaymentProps {
+export interface PaymentProps extends ReactStripeElements.InjectedStripeProps {
   order: Payment_order
+  relay?: RelayRefetchProp
+  router: Router
```

Comment:
What is this used for?

Reply:
It's used to move on to the review step: https://github.com/artsy/reaction/pull/1221/files#diff-ec0ceeeb68c615892b6b5a053119b0edR232

Reply:
Ah I see 👍